aws/resource_aws_workspaces_directory_test.go

```diff
@@ -66,6 +67,7 @@ func TestAccAwsWorkspacesDirectory(t *testing.T) {
 }
 
 func testAccAwsWorkspacesDirectory_basic(t *testing.T) {
+	var v workspaces.WorkspaceDirectory
```

Comment:
Good point 👍 I guess this should be added in a separate PR with the actual usage of `v` in checks. I know it's a good pattern to test TF state along with the actual resource one, however, it doesn't make sense without those tests.